index.html

```diff
@@ -51,23 +54,41 @@
                     <!--Cart-->
                 </ul>
             </div>
+            
+        <!-- START OUR POP-UP -->
+            <div class="pop-up-wrapper active" id="pop-up-wrapper">
+                <!--Creates the popup body-->
+                <div class="popup-overlay active-popup">
+                    <!--Creates the popup content-->
+                    <div class="popup-content active-popup">
+                        <div class = "popup-photo">
+                            <img src="img/wine.PNG">
+                        </div>
+                        <div class = "popup-text">
+                            <h1>NEVER MISS A NEW RELEASE</h1>
+                            <br>
+                            <p>We'd love to stay in touch with you and send a postcard from our Planet Winery time to time. Join our list to stay up to data on all things Planet Winery</p>
+                         </div>
+                        <!--popup's close button-->
+                        <div class = closeButton>
+                            <button class="close close-button-layout"><i class="fas fa-times"></i></i></button>
+                        </div>
```

Comment:
I wonder if we want to deal with styling of the popup, the close button feels too close to top right.